src/attributes.jl

```diff
@@ -184,6 +191,13 @@ end
 
 ## Instance attributes
 
+"""
+    ListOfInstanceAttributesSet()
+
+A `Vector{AbstractInstanceAttribute}` of all instance attributes that were set to the instance.
```

Comment:
How is a solver supposed to decide if an attribute was set, does it need to store a bool for every attribute it supports?

Reply:
I do not see any way around that. most solver specific attributes have default values but they should return these attributes. For instance if we have an MOI attributes for loglevel, if we do not set the log level of `src`, it should not impose its log level of `dest`.

Reply:
Ok, fair enough.